src/components/color-picker/RuiColorHue.vue

```diff
@@ -0,0 +1,127 @@
+<script lang="ts" setup>
+import { clamp } from '@vueuse/shared';
+import { roundTwoDecimal, useElementDrag } from './utils';
+
+const props = defineProps<{
+  modelValue: number;
+}>();
+
+const emit = defineEmits<{
+  (e: 'update:model-value', hue: number): void;
+}>();
+
+const barElement: Ref<HTMLElement | null> = ref(null);
+
+const modelValue = useVModel(props, 'modelValue', emit);
```

Comment:
does this send the proper event? I can see in the other place we had to specify the kebab-case event specifically.

Reply:
yes it works, at least in vue 3, I think `@update:model-value` just the same with `@update:modelValue`.
If I emit `update:modelValue` or `update:model-value`, both `@update:modelValue` and `@update:model-value` will be able to catch it.